python/chronos/test/bigdl/chronos/forecaster/test_autoformer_forecaster.py

```diff
@@ -270,8 +271,7 @@ def test_autoformer_forecaster_confidence_interval_with_loader(self):
                                           output_feature_num=2,
                                           label_len=12,
                                           freq='s',
-                                          seed=0,
-                                          moving_avg=20) # even
+                                          seed=0)
```

Comment:
why are we removing this even moving_avg test?

Reply:
even moving_avg is test in test_autoformer_forecaster_even_kernel. So this test is probably not needed here.